Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.h

```diff
@@ -97,5 +103,7 @@ namespace AzToolsFramework
         AZ::EntityComponentIdPair m_ownerEntityComponentId;
 
         AzFramework::PaintBrush m_paintBrush;
+
+        AZStd::unique_ptr<EditorPickEntitySelectionHelper> m_pickEntitySelectionMode;
```

Comment:
suggestion: You could potentially use `AZStd::optional` here instead of making this a pointer, it would work almost exactly the same it's just you don't then have the pointer indirection as the instance in a normal class member

Reply:
It still _looks_ like you have a pointer indirection though... Happy to change it, it's really half dozen of one vs the other to me.

Reply:
Yeah it's not a huge deal just in certain situations can be a performance win